data_diff/diff_tables.py

```diff
@@ -247,8 +272,7 @@ def _bisect_and_diff_tables(self, table1, table2, level=0, max_rows=None):
 
     def _diff_tables(self, table1, table2, level=0, segment_index=None, segment_count=None):
```

Comment:
While you're in here, could you output `key-range` and `size` for the full table above too? I think it's OK to wait for that `min/max` on both ends first.